### DISCUSSION TITLE
路由器reset之后，需要手动修改IP才能访问路由

### DISCUSSION THREAD

**Discussion by stwen3399** [coolsnowwolf/lede#9570] [R68S / 电犀牛专区]
路由器reset之后，用默认IP登录不了后台，查看Windows设备之后，显示未分配网关
![image](https://user-images.githubusercontent.com/107385695/173480078-944bcb61-d6b0-486a-b5c8-205de9323298.png)
![image](https://user-images.githubusercontent.com/107385695/173480124-7662014e-8512-4383-b7fb-b3dd6c73c356.png)
需要手动设置192.168.1.2or192.168.100.2（看固件版本）才能访问。而且有时候需要修改LAN口的静态地址之后也不能访问，Windows显示之前的网关，插拔网线之后也是不变，希望下一版固件可以路由器reset之后，不用手动修改IP访问路由。这是我目前遇到的问题，谢谢。

**Comment by SLiang-liao** [comment]
一样的错误

**Comment by 45142918** [comment]
一样的错误

**Comment by jhtao** [comment]
一样的错误
